packages/office-ui-fabric-react/src/components/Pivot/Pivot.tsx

```diff
@@ -144,15 +145,15 @@ export class Pivot extends BaseComponent<IPivotProps, IPivotState> {
         ariaLabel={ link.ariaLabel }
         role='tab'
         aria-selected={ this.state.selectedKey === itemKey }
-        name={ link.linkText }
+        name={ link.headerText }
       >
         { linkContent }
       </CommandButton>
     );
   }
 
   private _renderLinkContent = (link: IPivotItemProps): JSX.Element => {
-    const { itemCount, itemIcon, linkText } = link;
+    const { itemCount, itemIcon, headerText } = link;
```

Comment:
This is a breaking change. If I'm using the Pivot with "linkText", my pivot will no longer work. Can you adjust this to make sure it still works?

When you deprecate something, it means you need for it to still work with both old AND new. That way a partner can have time to adjust, but doesn't need to adjust now.

It would be very upsetting for partners to update Fabric, only to see no build errors, and then find out after the build that their Pivots have no text in them...